src/semidiscretization/semidiscretization_hyperbolic.jl

```diff
@@ -314,7 +314,7 @@ function Base.show(io::IO, ::MIME"text/plain", semi::SemidiscretizationHyperboli
 
         summary_line(io, "source terms", semi.source_terms)
         summary_line(io, "solver", semi.solver |> typeof |> nameof)
-        summary_line(io, "total #DOFs per field", ndofs(semi))
+        summary_line(io, "total #DOFs per field", ndofsglobal(semi))
```

Comment:
Please also make sure here that there are no rank-local quantities printed here

Reply:
Thanks for the hint!